### ISSUE TITLE
[Card] Card component static not changing color

### ISSUE THREAD

**Issue by leticiafontoura** [carbon-design-system/carbon-for-ibm-dotcom#6737]
By removing the `light: true` property for the card static, it remains with the light background, not changing to the grayish one.

**Comment by ariellalgilmore** [comment]
Hi @leticiafontoura, is there a link where we can see this happening? Here is a [codesandbox example](https://codesandbox.io/s/fervent-leaf-q2k2o?file=/index.html) of the static card without the `color-scheme="light"`.

**Comment by leticiafontoura** [comment]
Sorry for my lack of details.

This is for the react component. 
![image (15)](https://user-images.githubusercontent.com/47862337/127228646-efe90feb-5b82-4489-bb28-c5a82070a6e5.png)

```
const card = {
    heading: "Bla Bla",
    copy: "Copsssy",
    border: true,
    cardStatic: true,
    eyebrow: "dr. blabla",
    cta: {
      href: "",
      copy: ""
    },
  }
...
<Card {...card} />
```

**Comment by RobertaJHahn** [comment]
@leticiafontoura  What application are you using this component in?  Do you have a time frame for this bug to be fixed?

**Comment by leticiafontoura** [comment]
@RobertaJHahn no, I don't have a time frame really. the application is just on my computer atm but i'm using the nextJS template

**Comment by RobertaJHahn** [comment]
@leticiafontoura Good to know.  We are currently scheduling work several sprints out, so if there is no urgency, the work will be scheduled as the engineering resources become available.

**Comment by RobertaJHahn** [comment]
@leticiafontoura The dev work is finished and will be included in sprint v1.25.0.  The release date is Sept 28.

cc:  @NSDrowned
